packages/tools/pluggable-widgets-tools/src/typings-generator/generatePreviewTypes.ts

```diff
@@ -1,6 +1,7 @@
-import { Property } from "./WidgetXml";
 import { capitalizeFirstLetter, extractProperties } from "./helpers";
 
+import { Property } from "./WidgetXml";
```

Comment:
Nit: this import has unnecessarily changed

Reply:
Sorry, forgot to turn off the import extension in vscode again 😅

Reply:
Fixed :-)